extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -877,7 +887,9 @@ public static void addCookieHeader(@Nonnull final Map<String, List<String>> head
 
     @Nonnull
     public static String generateConsentCookie() {
-        return CONSENT_COOKIE + 100 + numberGenerator.nextInt(900);
+        return "CONSENT=YES+";
+        // Doesn't work
+        // return CONSENT_COOKIE + numberGenerator.nextInt(900);
```

Comment:
I tested with YoutubeStreamExtractorDefaultTest -> DescriptionTestPewdiepie, YoutubeChannelExtractorTest -> Gronkh and it works.

Reply:
I will continue the discussion there